Sources/App/Models/Build.swift

```diff
@@ -32,6 +32,7 @@ final class Build: Model, Content {
     @Field(key: "job_url")
     var jobUrl: String?
 
+    @available(*, deprecated)
```

Comment:
Marking the `logs` field deprecated for now. Will ditch the whole column in a follow up PR once we're happy with the result.